uaclient/cli/formatter.py

```diff
@@ -133,11 +133,12 @@ def _get_column_sizes(self) -> List[int]:
         return column_sizes
 
     def __str__(self) -> str:
+        rows = self.rows
```

Comment:
why do we need the `rows` variable if the fix is applied on `new_rows.append` call ?

Reply:
Because the new_rows thing is only done if the table exceeds the line length. If there is no line exceeding the available space, then the whole thing short-circuits to using the actual content in self.rows, without change.